package.json

```diff
@@ -37,5 +38,8 @@
     "web-ext": "^2.4.0",
     "webpack": "^3.10.0"
   },
-  "license": "MPL-2.0"
+  "license": "MPL-2.0",
+  "dependencies": {
```

Comment:
So this is a minor thing, but can we add `crx` to `devDependencies` instead of `dependencies`? 

`dependencies` are things that are required during runtime, and `devDependencies` are things required to develop, or build.

(the typical way to add something as a devDependency to a package.json is via `npm install --save-dev foo`)